hwdb.d/60-keyboard.hwdb

```diff
@@ -620,6 +620,14 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPZBook*:*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHPZBook*:*
  KEYBOARD_KEY_81=f20                                    # Fn+F8; Microphone mute button, should be micmute
 
+# HP ZBook Studio G5
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHPZBookStudioG5*:*
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPZBookStudioG5*:*
```

Comment:
Why are there two match lines? You wrote "I tested this on my ZBook Studio G5", i.e. just a single machine, iiuc.

Reply:
I only tested the svnHP one, but since other HP laptops had vendor Hewlett-Packard as well I figured it couldn't hurt to add it.

Reply:
Is it better to remove this line?

Reply:
Yeah, I don't think we should add a line that doesn't match hardware that we have actually seen.

Reply:
Allright, I've removed the line.